Adding definition for a frame (#7993)
flutter/website: src/development/tools/devtools/performance.md

* Timeline events chart
* CPU profiler

{{site.alert.secondary}}
  **Use a profile build of your application to analyze performance.**

>   Frame rendering times aren't indicative of release performance
>   unless your application is run in profile mode.

Reviewer: **suggestion:** Remove the nominalization.
```suggestion
  Don't take frame rendering times to equal release performance
  unless you run your app in profile mode.
```

Author: Even in profile mode, performance isn't the same as release mode. This is misleading.

{{site.alert.end}}

The performance view also supports importing and exporting of
data snapshots. For more information,
check out the [Import and export][] section.

## What is a frame in Flutter?

> Flutter is designed to render its UI at 60 frames per second
> (fps), or 120 fps on devices capable of 120Hz updates.
> This means that, approximately every 16ms, the UI updates
> to reflect animations, or other changes to the UI. A frame
> that takes longer than 16ms to render can cause jank 
> (jerky movement).

Reviewer: **issue:** The graf doesn't answer the question posed in the heading.
```suggestion
Flutter renders its UI at least 60 times each second.
Each render is called a _frame_. Flutter can render 
120 frames per second (fps) on displays that refresh at 120Hz.
This means Flutter updates the UI about every 16ms
to reflect animations or other changes to the UI. If a frame
renders in more than 16ms, it can cause jank, or jerky movement,
on the device display.
```

Author: I wouldn't say "it renders" as opposed to "it's designed to render". In other words, it MIGHT take longer, in which case it's the programmers fault.

## Flutter frames chart

This chart contains Flutter frame information for your application.
Each bar set in the chart represents a single Flutter frame.
The bars are color-coded to highlight the different portions
of work that occur when rendering a Flutter frame: work from

> the UI thread and work from the raster thread (previously known
> as the GPU thread).

Reviewer: **suggestion:** Cut this sentence into two to make it more concise.
```suggestion
the UI and raster threads.
_Raster thread_ is the new term for the GPU thread.
```

Author: Again, it's not a "new" term. This change happened years ago. I leave it because it can still be useful for some.

see [Flutter performance profiling][].

### Shader compilation

Shader compilation occurs when a shader is first used in your Flutter
app. Frames that perform shader compilation are marked in dark
red:

![Screenshot of shader compilation for a frame]({{site.url}}/assets/images/docs/tools/devtools/shader-compilation-frames-chart.png)

> For more information on how to reduce shader compilation jank,

Reviewer: **quibble:** `For more information` lacks agency or a call to action.
```suggestion
To learn how to reduce shader compilation jank,
```
